app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -375,7 +394,7 @@ public void onEventsDownloadDone(EventDownloadEvent event) {
             }
         } else {
 
-            downloadFailed();
+//            downloadFailed();
```

Comment:
I'm still in the process of migrating the code to fit the new views etc used by the new design